### ISSUE TITLE
Index on CPAN out of sync

### ISSUE THREAD

**Issue by andk** [iamalnewkirk/data-object#169]
There are 21 modules still indexed in the CPAN index that seem to have been downgraded from 0.61 to undef at some point in the past. Here the names:

1. Data::Object::Array
2. Data::Object::Class
3. Data::Object::Code
4. Data::Object::Exception
5. Data::Object::Float
6. Data::Object::Hash
7. Data::Object::Integer
8. Data::Object::Library
9. Data::Object::Number
10. Data::Object::Regexp
11. Data::Object::Role
12. Data::Object::Role::Detract
13. Data::Object::Role::Dumper
14. Data::Object::Role::Output
15. Data::Object::Role::Proxyable
16. Data::Object::Role::Throwable
17. Data::Object::Role::Type
18. Data::Object::Scalar
19. Data::Object::Signatures
20. Data::Object::String
21. Data::Object::Undef

It's not a big deal, but sure could be resolved somehow. It leads to wrong suggestions when asking the "wrong" question:
```
cpan[4]> m Data::Object::Integer
Module id = Data::Object::Integer
    CPAN_USERID  AWNCORP (Al Newkirk <awncorp@cpan.org>)
    CPAN_VERSION 0.61
    CPAN_FILE    A/AW/AWNCORP/Data-Object-0.61.tar.gz
    INST_FILE    (not installed)
```
Or when use_sqlite is in effect:
```
cpan[1]> m Data::Object::Integer
Database was generated on Mon, 25 Mar 2019 04:06:15 GMT
Updating database file ... Done!
No objects of type Module found for argument Data::Object::Integer
```
Also with cpanm, one might install the outdated version:
```
cpanm Data::Object::Integer
--> Working on Data::Object::Integer
Fetching http://www.cpan.org/authors/id/A/AW/AWNCORP/Data-Object-0.61.tar.gz ... OK
Configuring Data-Object-0.61 ... OK
```

Maybe just removing the version 0.61 from the cpan would be an option, or starting again with numbering the modules. Or there is also a page on PAUSE that would allow you to unindex historic versions: https://pause.perl.org/pause/authenquery?ACTION=reset_version

I'm a PAUSE admin, let me know when you think we can help.

Regards,

**Comment by iamalnewkirk** [comment]
Thanks for this report and detailed explanation. I'm resolving the issues now and will release an update shortly thereafter.

**Comment by iamalnewkirk** [comment]
Fixed in #170
